Week-2/animation.css

```diff
@@ -0,0 +1,74 @@
+.main {
```

Comment:
@rajeshvlega9 Animation looks good. Nice work. Can you format the css, it looks not readable

Reply:
formatted the animation.css with prettier now